CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -125,6 +193,11 @@ function PlayerHospital:dailyAdviceChecks()
     end
   end
 
+  -- Warn player of no handymen if there are plants
+  if day == 18 and self:countPlants() > 0 then
+    self:msgPlant(false)
+  end
```

Comment:
Add a no-handyman check.

EDIT: I wrote the above line as the function name doesn't imply it takes handy men into account in the message.
change the function name instead.

The name of the function should cover what it does so one can judge correctness of a line with a function call without knowing the specifics of the function internals.